### PR TITLE
Unify Combat Follow-through and Combat Momentum triggers

### PR DESCRIPTION
**Pull request by oconnor0**

### CODE REVIEW

File: core/SRD.md [hunk 1]
```diff
@@ -1170,14 +1170,9 @@ attribute in place of Persuasion for the action roll.
 
 -   **Tier 1:** None
 
-**Description:** You are able to decimate many enemies in quick succession, like a
-legendary Samurai warrior or matchless elven archer. \newline
+**Description:** You are able to decimate many enemies in quick succession. \newline
```
Comment (brianfeister) [inline core/SRD.md]: Why are you removing flavor text? No bueno

Comment (oconnor0) [inline core/SRD.md]: Sorry. I thought you wanted flavor text pulled out of the SRD.

**Comment by oconnor0** [comment]
The flavor text for Combat Follow-through and Combat Momentum should be back in the SRD.

**Comment by brianfeister** [comment]
"Flavor text not in the SRD" just means that the non-mechanical segues that explain the rules via example are omitted from the SRD. The entire text of the rules for banes, boons, feats, perks, and flaws should be identical in all cases between core and the SRD

**Comment by oconnor0** [comment]
Ah, thank you for clarifying. I misunderstood.I put flavor back in. Are they what you need/want?

**Comment by brianfeister** [comment]
Yup, looks good. Much appreciated 👍  💯

**Pull request merged**